layouts/_default/main.html

```diff
@@ -1,10 +1,14 @@
+{{ define "title" }}
+  <title>{{ .Param "title-tag" | default .LinkTitle }}</title>
+{{ end }}
```

Comment:
This remove "• Spotlight PA" from the page title. Why override this block? Default should be fine.